bin/index.test.js

```diff
@@ -16,10 +16,8 @@
 const { readFileSync } = require('fs');
 
 test('Verify that version from package.json matches version in the code', () => {
-  const packageJson = readFileSync('package.json', 'utf8');
-  const packageVersion = JSON.parse(packageJson).version;
   const testRegEx = new RegExp(`VERSION = '([0-9.]+)'`);
   const indexJs = readFileSync('bin/index.js', 'utf8');
   const indexJsVersion = indexJs.match(testRegEx);
-  expect(indexJsVersion[0]).toBe(`VERSION = '${packageVersion}'`);
+  expect(indexJsVersion[0]).toBe(`VERSION = '${require('../package.json').version}'`);
```

Comment:
I'm not much of a fan of cramming so much into a single line... To improve readability, would you mind assigning the the pkg version to a variable and then using the variable in the expect statement?

Reply:
Sure , not a problem. Also the main idea is to avoid a fs read and a JSON.parse :+1:

Reply:
Ahhhh... I find that much easier to read. Thank you!